src/libostree/ostree-repo.c

```diff
@@ -2094,6 +2094,27 @@ append_remotes_d (OstreeRepo          *self,
 }
 
 gboolean
+ostree_repo_exists (OstreeRepo    *self,
+                    gboolean      *does_exist,
+                    GError        **error)
+{
+  struct stat stbuf;
+
+  if (lstat (gs_file_get_path_cached (self->objects_dir), &stbuf) < 0)
```

Comment:
I would rename this one to `is_valid()`. Actually, `is_created()` would be more appropriate I suppose since it's `create()` that does the work.